modules/nix-bitcoin.nix

```diff
@@ -4,6 +4,7 @@ with lib;
 
 let
   cfg = config.services.nix-bitcoin;
+  electrs-cfg = config.services.electrs;
```

Comment:
What is the point of electrs-cfg?

Reply:
This is just a matter of style, I prefer using `electrs-cfg` instead of `config.services.electrs` as it is shorter and keeps the line length shorter also.